third_party/zip/zipcloak.c

```diff
@@ -634,19 +634,19 @@ int main(argc, argv)
 
     /* Get password */
     if (getp("Enter password: ", passwd, IZ_PWLEN+1) == NULL)
-        ziperr(ZE_PARMS,
+        ziperr(ZE_PARAMS,
                "stderr is not a tty (you may never see this message!)");
 
     if (decrypt == 0) {
         if (getp("Verify password: ", verify, IZ_PWLEN+1) == NULL)
-               ziperr(ZE_PARMS,
+               ziperr(ZE_PARAMS,
                       "stderr is not a tty (you may never see this message!)");
 
         if (strcmp(passwd, verify))
-               ziperr(ZE_PARMS, "password verification failed");
+               ziperr(ZE_PARAMS, "password verification failed");
```

Comment:
Could you revert the change to `PARMS`? It changes too much to be a typo.

Reply:
I appreciate all your input across these files. I'll update everything accordingly